app/views/declaration/draft_confirmation_page.scala.html

```diff
@@ -28,17 +30,26 @@
         <h1 class="heading-xlarge">
             @messages("confirmation.draft.title")
         </h1>
-        <p class="font-large">@messages("confirmation.draft.info", flash.get(FlashKeys.expiryDate).getOrElse("-"))</p>
+        <p class="font-large">
+         @messages("confirmation.draft.info")
```

Comment:
That message key needs the {0} param removing I think.     You've also increased the separation between the message and the date by introducing a new paragraph.  Why not <p> Message <br/> Date </p> ?

Reply:
I removed the `{0}` from the message and made it into two lines so the date is always on the same line.

Not sure `<br/>` gives us anything over just having two `<p>` tags, unless one is more accessible?

Reply:
Missed the change to message.en.    Difference is single line spacing.